src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -55,12 +61,12 @@ const CourseEnrollments = ({ children }) => {
     const assignmentsData = sortAssignmentsByAssignmentStatus(data);
     setAssignments(assignmentsData);
 
-    const hasCancelledAssignments = assignmentsData?.some(
+    const isCancelledAssignments = assignmentsData?.some(
       assignment => assignment.state === ASSIGNMENT_TYPES.CANCELLED,
     );
+    setHasCancelledAssignments(isCancelledAssignments);
     const hasExpiredAssignments = assignmentsData?.some(assignment => isAssignmentExpired(assignment));
```

Comment:
Do we want to set the same behavior for expired assignments?

Reply:
Good question. Based on the [Figma](https://www.figma.com/file/8da9TyEpEzgxQld3DZE67w/LC2---Top-down-Assignment?type=design&node-id=2545-100329&mode=dev) mocks regarding the "Dismiss" behavior on the dashboard, I believe its intended to be consistent between cancelation and the expired cases.

<img width="911" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/f32e3e1a-b749-4272-b5b4-0fdb404bb4f6">